cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/namespace/DistributedStorageProviderNamespaceAdmin.java

```diff
@@ -106,8 +106,9 @@ public void create(NamespaceMeta namespaceMeta) throws IOException, ExploreExcep
 
     try (HBaseAdmin admin = new HBaseAdmin(hConf)) {
       if (!tableUtil.hasNamespace(admin, hbaseNamespace)) {
-        throw new IOException(String.format("Custom mapped HBase namespace doesn't exist %s for namespace %s",
-                                            hbaseNamespace, namespaceMeta.getName()));
+        throw new IOException(String.format("HBase namespace '%s' specified for custom mapped namespace '%s' does not" +
```

Comment:
can we remove the words "custom mapped"? Its a very developer-ish term, that users may not understand. It doesn't even matter in this context right? Removing it completely should also work.